test/unit/smartapp-spec.js

```diff
@@ -36,34 +36,15 @@ describe('smartapp-spec', () => {
 			installData: expectedData,
 			settings: {}
 		})
-		assert.strictEqual(receivedData, expectedData)
-	})
 
-	it('should handle UNINSTALL event', () => {
```

Comment:
This test was duplicating what was already covered in an [existing suite](https://github.com/SmartThingsCommunity/smartapp-sdk-nodejs/blob/73f9ea0f9e0d521e3cb78b6bc93ae5797be06405/test/unit/events/uninstalled-event-spec.js#L4).